extensions/amp-story/1.0/amp-story.js

```diff
@@ -1253,6 +1263,24 @@ export class AmpStory extends AMP.BaseElement {
     activePage.previous();
   }
 
+  /**
+   * Handles EventType.NO_PREVIOUS_PAGE events.
+   * @private
+   */
+  onNoPreviousPage_() {
+    if (this.viewer_.hasCapability('swipe')) {
+      this.viewer_./*OK*/ sendMessage(
```

Comment:
Perhaps in a separate PR: Don't we need to handle a response (here and for no next page)?  e.g. if the viewer tells us "there is no previous story" when the user has tapped to previous, we should still show the first page hint?